### DISCUSSION TITLE
When minimizing a file in a pull request after reviewing it, the "Expand Up" tooltip always come up if you hover over the expand element blocking the clicking of the minimize.

### DISCUSSION THREAD

**Discussion by zhiyi-zhang-duke** [community/community#57042] [Pull Requests]
### Select Topic Area

Bug

### Body

Screenshot example:
![image](https://github.com/community/community/assets/68999131/6cc6a92e-43f0-4395-bd80-0cb7ef7bb8fc)
This tooltip is mostly unnecessary and adds 2-3 extra seconds to each time a user wants to minimize a file. If you attempt to click on minimize anyway, nothing happens as you're just clicking on the tooltip.

Suggestion:
I would expand the tooltip downwards. It doesn't completely cover a different ui element and it preserve the intention of the tooltip.

**Comment by zbjornson** [comment]
This is a duplicate of #54674.
